wiki/Tournaments/GTS/BGTS_2020/en.md

```diff
@@ -0,0 +1,216 @@
+| Designer | ![][flag_PL] [E-M-i](https://osu.ppy.sh/users/9148286), ![][flag_PH] [OsuMe65](https://osu.ppy.sh/users/852867), ![][flag_KR] [POCARI SWEAT](https://osu.ppy.sh/users/5082685) |
+| Recruiter | ![][flag_GB] [B0tch3d](https://osu.ppy.sh/users/9864847), ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263), ![][flag_SG] [kuudere-desu](https://osu.ppy.sh/users/10694200), ![][flag_IT] [LordEnder](https://osu.ppy.sh/users/4609767), ![][flag_ID] [Reed\_405](https://osu.ppy.sh/users/9965069), ![][flag_HK] [-Storm7-](https://osu.ppy.sh/users/12248285), ![][flag_US] [Swovine](https://osu.ppy.sh/users/6666316), ![][flag_FR] [TLQ\_Yoshii](https://osu.ppy.sh/users/7157133), ![][flag_AR] [Vaf](https://osu.ppy.sh/users/12589048), ![][flag_CN] [WonderMan](https://osu.ppy.sh/users/2221895) |
+| Translator | ![][flag_CL] [Catulus](https://osu.ppy.sh/users/6276709), ![][flag_SG] [dantoh000](https://osu.ppy.sh/users/11414361), ![][flag_IT] [LordEnder](https://osu.ppy.sh/users/4609767), ![][flag_DE] [Sayira](https://osu.ppy.sh/users/7253958), ![][flag_HK] [-Storm7-](https://osu.ppy.sh/users/12248285), ![][flag_NL] [TaikoMom](https://osu.ppy.sh/users/9086438), ![][flag_FR] [TLQ\_Yoshii](https://osu.ppy.sh/users/7157133) |
+| Developer | ![][flag_US] [Cychloryn](https://osu.ppy.sh/users/6921736), ![][flag_NL] [oliebol](https://osu.ppy.sh/users/2756335), ![][flag_FR] [ThePooN](https://osu.ppy.sh/users/718454) |
+| Wiki editor | ![][flag_ID] [fajar13k](https://osu.ppy.sh/users/7100002) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1178018)
+- [GTS Discord server](https://discord.gg/3mGC3HB)
+- [GTS website](https://gtsosu.com/)
+- Livestream
+  - [GTSosu](https://www.twitch.tv/gtsosu)
+  - [GTSosu\_b](https://www.twitch.tv/gtsosu_b)
+
+## Participants
+
+*Participants will be listed as soon as screening phase has been done by the osu! Officials.*
+
+## Mappools
+
+*Mappools will be listed as soon as the first stage kicks off.*
+
+## Ruleset
+
+### General rules
+
+1. Beginner Global Taiko Showdown is a **2v2 double-elimination tournament** with teams of **5 players maximum**.
+2. The teams will first go through a **Qualifiers**. Then they will play in a **Group Stage** and finally play in a **double-elimination bracket**.
+3. The scoring system that will be used is **ScoreV2**.
+4. Any player who meets the requirements stated in the [rank restrictions](#rank-restrictions) section is allowed to sign up. Since AGTS will aim to get badged, some players might be declined by the osu! staff (via screening).
+5. To keep the competition fair, there is a team in charge of checking the profile of every single player registered, and if a player's skill is too high for this tournament, they'll not be able to play in this tournament.
```

Comment:
```suggestion
5. Players may have their registration denied if they are deemed to be too skilled for BGTS.
```
think that's all you need to say here, the staff section is somewhere else and this is the "rule". this may fit better in "Tournament registration and teams creation" tho